src/diffusers/models/controlnet_sd3.py

```diff
@@ -99,7 +103,7 @@ def __init__(
             height=sample_size,
             width=sample_size,
             patch_size=patch_size,
-            in_channels=in_channels + extra_conditioning_channels,
+            in_channels=in_channels,
```

Comment:
Is this change not related to SD 3.5?

Reply:
It is releated to whether the control net is used for inpainting. 
The control image for inpainiting has additional mask channel, but the normal controlnet does not need it. [src/diffusers/pipelines/controlnet_sd3/pipeline_stable_diffusion_3_controlnet_inpainting.py](https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/controlnet_sd3/pipeline_stable_diffusion_3_controlnet_inpainting.py#L746)

Reply:
why are we removing this then? we can set it to 0 when it is not needed?

Reply:
Sorry, you are right.  I cancelled the commit and I changed the  the default num_extra_conditioning_channels to 0 in the from_transformer method such that when using [script](https://github.com/huggingface/diffusers/blob/main/examples/controlnet/train_controlnet_sd3.py), erros will not occur.